customize-assets/tutorial.md

```diff
@@ -0,0 +1,69 @@
+
+# Pryv Customize assets tutorial
+
+In this tutorial we describe the procedure to customize your Pryv.io web apps. As explained in the [README](customize-assets/README.md), it is also possible to directly edit your platform assets in the **public_html/assets** folder of your Open Pryv.io instance.
+
+However, when working in a production environment, we stronly recommend to follow the steps below to change assets for your Open Pryv.io platform.
+
+This will allow you to benefit from updates and improvements of our [Javascript library](https://github.com/pryv/lib-js) and [Pryv.io visual assets](https://github.com/pryv/assets-open-pryv.io).
```

Comment:
allow you to keep track of your changes and our updates to the [Pryv.io visual assets](https://github.com/pryv/assets-open-pryv.io).